udata/core/dataset/models.py

```diff
@@ -595,6 +595,14 @@ class Dataset(Auditable, WithMetrics, DatasetBadgeMixin, Owned, Linkable, db.Doc
         ),
         auditable=False,
     )
+    last_update = field(
+        DateTimeField(
+            verbose_name=_("Last update of the dataset resources"),
+            default=datetime.utcnow,
+            required=True,
```

Comment:
I've added a default since the field is required (similar to other datetime fields)